pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -90,6 +90,8 @@ service ImageService {
     // This call is idempotent, and must not return an error if the image has
     // already been removed.
     rpc RemoveImage(RemoveImageRequest) returns (RemoveImageResponse) {}
+    // ImageFsInfo returns filesytem information of images.
```

Comment:
typo: s/filesytem/filesystem

// ImageFSInfo returns information of the filesystem that is used to store images.